### DISCUSSION TITLE
No MacBook Support

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#566] [General]
I visited my local Walmart the other day, and they had a MacBook on display. I went to the App Store on there, and I heard some of the iPhone and iPad apps are compatible with MacBook. Unfortunately, ToonSquid is not one of those compatible apps, and it's a shame, too, because it's the best animation app I've ever used, and it'd be the first one I'd get if I ever got a MacBook. Could MacBook compatibility come in a future update?

**Comment by Cat-Butler** [comment]
Desktop compatibility has been [marked as a future possibility for M(n)-based CPUs](https://toonsquid.com/handbook/guides/faq/#platforms), but will probably require a significant chunk of developer time (e.g. enabling touch alternatives, handling different screen resolutions, handling non-full-screen display and resizable windows, possible multiple instances/documents running). That's time away from developing more animation-related features. Graphics-style apps are also, I believe (from what I've read of the developer documentation), more difficult to make compatible than text-style apps (buttons and input fields, etc).

It's also going to potentially be competing with a fairly saturated desktop market for existing 2D animation apps, many of which already have impressive professional feature lists  (Adobe Animate, Toon Boom Harmony, Moho, Cacani, CelAction2D, TVPaint, Cartoon Animator, not to mention 2D animation features in Clip Studio Paint and FOSS alternatives such as Blender Grease Pencil, Krita, OpenToonz, Synfig, and probably plenty more that haven't occurred to me immediately).
I will note that, apart from the FOSS apps, most of these apps also come with a professional price.

I'd also like to have ToonSquid on macOS just for the possibility of combining it with expanded workflows and tools (so here's hoping it eventually happens), but I'll admit on the desktop Moho holds my personal preference.

**Comment by DWrightAnimation** [comment]
Just bumping this up, but how likely would a Mac version of ToonSquid be? Not even Procreate, Callipeg, or Procreate Dreams work on Mac.

**Reply by Rcomics** [reply]
Hey update on that, procreate is actually releasing on Mac! technically all apps CAN run on Mac but it's up to devs, Keiwan said he would want to refine the app before porting it, that's why he didn't enable the basic Mac porting
